test/test_suite.py

```diff
@@ -1645,6 +1632,43 @@ def test_offset_only(self):
                 list(connection.execute(self._table.select().offset(offset)).fetchall())
 
 
+class ExecutionOptionsStalenessTest(fixtures.TestBase):
+    """
+    Check that `execution_options()` method correctly
+    sets parameters on the underlying DB API connection.
+    """
+
+    def setUp(self):
+        self._engine = create_engine(get_db_url(), pool_size=1)
+        metadata = MetaData(bind=self._engine)
+
+        self._table = Table(
+            "execution_options",
+            metadata,
+            Column("opt_id", Integer, primary_key=True),
+            Column("opt_name", String(16), nullable=False),
+        )
+
+        metadata.create_all(self._engine)
+        time.sleep(1)
```

Comment:
As we're using `staleness`, and the emulator works really fast, we're getting in a situation when there is no test table yet, but we're already trying to read it, getting an error. Doing a small pause to avoid it.